recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "anaconda-cloud-auth" %}
-{% set version = "0.1.4" %}
+{% set version = "0.4.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anaconda_cloud_auth-{{ version }}.tar.gz
-  sha256: f0f029d7c87dc86ff15c005d2387d102a56d861ec64559a734dd032103d768bf
+  sha256: ff61aab300fa174a4ee02578e22be81ee1e093a9aa75d835e232130ac75894df
 
 build:
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```

Comment:
I'm not sure, but you might need to include `wheel`, and `setuptools` rather than `setuptools-scm`. Some patterns I've seen in other feedstocks. although you'd need to check with someone more familiar with this

Reply:
We don't need setuptools because the package uses hatchling. But `setuptools-scm` in theory shouldn't be needed because they use `hatch-vcs` to get the version, not `setuptools-scm`, see https://github.com/anaconda/anaconda-cloud-tools/blob/anaconda-cloud-auth-v0.4.1/libs/anaconda-cloud-auth/pyproject.toml#L58-L59. But they do say "setuptools-scm" in https://github.com/anaconda/anaconda-cloud-tools/blob/anaconda-cloud-auth-v0.4.1/libs/anaconda-cloud-auth/pyproject.toml#L3, which is probably a leftover from a previous version I think.